src/transforms/interval.js

```diff
@@ -6,6 +6,10 @@ import {maybeInsetX, maybeInsetY} from "./inset.js";
 // chose between UTC and local time (or better, an explicit timeZone option).
 function maybeInterval(interval) {
   if (interval == null) return;
+  if (typeof interval === "number" && interval) {
```

Comment:
What’s the truthy `interval` test for? Testing for nonzero, non-NaN? I think I’d prefer to drop this because it’s not sufficient to ensure good behavior (for example, Infinity or -Infinity will produce NaN below), and it’s just simpler to avoid trying.

```suggestion
  if (typeof interval === "number") {
```

Alternatively, we could ignore NaN specifically:

```suggestion
  if (typeof interval === "number" && !isNaN(interval)) {
```

This will still break for zero and infinities, but those are at least considered defined values, so it’s probably reasonable that it breaks.

Reply:
I'm OK with not trying :)